File: Python/p11360.py
kasus = int(input())
for i in range(kasus):
    banyak = int(input())
    data = [[0 for x in range(banyak)] for y in range(banyak)]
    for j in range(banyak):
        temp = int(input())
        for k in range(banyak-1, -1, -1):
            data[j][k] = temp % 10
            temp = int(temp/10)

    bykComm = int(input())
    for j in range(bykComm):
        command = input().split()
        if command[0] == "row":
            for l in range(banyak):
                temp = data[int(command[1]) - 1][l]
                data[int(command[1]) - 1][l] = data[int(command[2]) - 1][l]
                data[int(command[2]) - 1][l] = temp

        elif command[0] == "col":
            for l in range(banyak):
                temp = data[l][int(command[1]) - 1]
                data[l][int(command[1]) - 1] = data[l][int(command[2]) - 1]
                data[l][int(command[2]) - 1] = temp

        elif command[0] == "inc":
            for k in range(banyak):
                for l in range(banyak):
                    data[k][l] += 1
                    if data[k][l] > 9:
                        data[k][l] %= 10

        elif command[0] == "dec":
            for k in range(banyak):
                for l in range(banyak):
                    data[k][l] -= 1
                    if data[k][l] < 0:
                        data[k][l] += 10

        else:
            data2 = [[0 for x in range(banyak)] for y in range(banyak)]
            for k in range(banyak):
                for l in range(banyak):
                    data2[l][k] = data[k][l]

            for k in range(banyak):
                for l in range(banyak):
                    data[k][l] = data2[k][l]

    print("Case #{:d}".format(i + 1))
    for j in range(banyak):
        res = ""
        for k in range(banyak):
            res += str(data[j][k])

        print(res)

    print()

File: Java/p11360.java

import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
/**
 *
 * @author JASONYEHEZKIEL
 */
public class p11360 {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int kasus = input.nextInt();
        for (int i = 1; i <= kasus; i++) {
            int banyak = input.nextInt();
            int[][] data = new int[banyak][banyak];
            for (int j = 0; j < banyak; j++) {
                int tempmasukan = input.nextInt();
                for (int k = 0; k < banyak; k++) {
                    data[j][k] = tempmasukan % 10;
                    tempmasukan /= 10;
                }
            }

            int bykcomm = input.nextInt();
            for (int j = 0; j < bykcomm; j++) {
                String comm = input.next();
                switch (comm.charAt(0)) {
                    case 'r':
                        int rowA = input.nextInt();
                        int rowB = input.nextInt();
                        rowA--;
                        rowB--;
                        for (int k = 0; k < banyak; k++) {
                            int temp = data[rowA][k];
                            data[rowA][k] = data[rowB][k];
                            data[rowB][k] = temp;
                        }
                        break;
                    case 'c':
                        int colA = input.nextInt();
                        int colB = input.nextInt();
                        colA--;
                        colB--;
                        for (int k = 0; k < banyak; k++) {
                            int temp = data[k][colA];
                            data[k][colA] = data[k][colB];
                            data[k][colB] = temp;
                        }
                        break;

                    case 'i':
                        for (int k = 0; k < banyak; k++) {
                            for (int l = 0; l < banyak; l++) {
                                data[k][l]++;
                                if (data[k][l] > 9) {
                                    data[k][l] %= 10;
                                }
                            }
                        }
                        break;

                    case 'd':
                        for (int k = 0; k < banyak; k++) {
                            for (int l = 0; l < banyak; l++) {
                                data[k][l]--;
                                if (data[k][l] < 0) {
                                    data[k][l] += 10;
                                }
                            }
                        }
                        break;

                    case 't':
                        int[][] temp = new int[banyak][banyak];
                        for (int k = 0; k < banyak; k++) {
                            for (int l = 0; l < banyak; l++) {
                                temp[l][k] = data[k][l];
                            }
                        }

                        for (int k = 0; k < banyak; k++) {
                            for (int l = 0; l < banyak; l++) {
                                data[k][l] = temp[k][l];
                            }
                        }
                        break;

                }
            }

            System.out.printf("Case #%d\n", i);
            for (int k = 0; k < banyak; k++) {
                for (int l = 0; l < banyak; l++) {
                    System.out.printf("%d", data[k][l]);
                }
                System.out.println();
            }

            System.out.println();
        }
    }
}